src/base/system_parameter.c

```diff
@@ -4490,7 +4490,7 @@ static SYSPRM_PARAM prm_Def[] = {
    (DUP_PRM_FUNC) NULL},
   {PRM_ID_READ_ONLY_MODE,
    PRM_NAME_READ_ONLY_MODE,
-   (PRM_FOR_SERVER | PRM_FOR_CLIENT),
+   (PRM_FOR_SERVER | PRM_USER_CHANGE | PRM_HIDDEN),
```

Comment:
Why do you change PRM_FOR_CLIENT to PRM_USER_CHANGE?

Reply:
That was my mistake. 

I just copied other lines that including "PRM_HIDDEN" because I try not to break the code style.

Should I close this PR and push again?

Reply:
@eido5 No. You just follow my guide.
1. fix it in the local
2. commit
3. push

After you have done this, you will be able to see that your modifications are shown in this PR.